ecosystem/typescript/sdk/src/utils/property_map_serializer.test.ts

```diff
@@ -0,0 +1,23 @@
+import { getPropertyValueRaw } from "./property_map_serializer";
+import { bcsSerializeBool, bcsSerializeStr, bcsSerializeU128, bcsSerializeU8, bcsSerializeUint64, Bytes } from "../bcs";
+import assert from "assert";
```

Comment:
Although assert works in tests, we should use Jest's assertion.

Reply:
done